src/Raven.Pal/src/posix/linuxonly.c

```diff
@@ -108,5 +114,179 @@ rvn_test_storage_durability(
     return rc;
 }
 
+bool _io_ring_supported()
+{
+  if(sizeof(void*) != 8)
+    return false;
 
-#endif
+   struct utsname buffer; 
+   if (uname(&buffer) != 0)
+        return false;
+
+  int curr_major = 0, curr_minor = 0, curr_patch = 0;
+  sscanf(buffer.release, "%d.%d.%d", &curr_major, &curr_minor, &curr_patch); 
```

Comment:
In Windows we dont need to use any parsing, this information is available on a bitmap. It seems a bit of a mismeanor to have to do parsing on every pager init. While it is not a soooo common operation, we could just store the value in the process and reuse it.

Reply:
Done